pyocm/pyocm.py

```diff
@@ -5,41 +5,46 @@
 
 class ocm_api:
     """."""
-    def __init__(self, api="https://api.openshift.com/api/", 
+    def __init__(self, api="https://api.openshift.com/api/",
             ca_path="/etc/pki/tls/certs/ca-bundle.crt", token=None):
 
         if token is None:
             raise Exception("Token was not supplied.")
 
+        self.thread_pool = ThreadPoolExecutor(4)
```

Comment:
This keeps issues with the `__del__` operation from showing up if `__init__` fails, and removes extra exceptions (that are erroneous).

Reply:
I'd put this above the `if token is None` check, since the __del__ function will still be called, and will fail since thread_pool won't exist.